src/main/java/com/onarandombox/MultiverseCore/MultiverseCore.java

```diff
@@ -202,6 +202,7 @@ private void registerEvents() {
      */
     private void registerCommands() {
         this.commandManager = new MVCommandManager(this);
+        this.commandManager.registerCommand(new RootCommand(this));
```

Comment:
IMO, the root command should go first because it breaks things when it's missing. It's meant to lead the subcommands, instead of blending in amongst them.

It might be nice to keep things this way in the event we register more commands (e.g., direct aliases such as `/mvcreate`).

Reply:
im not so sure about alias yet..

regarding the commands, i don't experience at issues with the ordering, its easier alphabetically bcu of how IDE will order all the command classes file alphabetically. So its easier to match and reference

Reply:
Alright, fair enough. I've moved it down.